src/components/FlowRunsAccordion.vue

```diff
@@ -56,4 +65,10 @@
 
     return flow
   }
-</script>
\ No newline at end of file
+</script>
+
+<style>
+.p-accordion__section:first-child .flow-runs-accordion-header:first-of-type { @apply
+  border-transparent
```

Comment:
Rather than turning it transparent let's remove it. I think it's border-0 or border-none